src/formatters/checkboxFormatter.tsx

```diff
@@ -50,27 +49,28 @@ const checkboxLabelDisabled = css`
 
 const checkboxLabelDisabledClassname = `rdg-checkbox-label-disabled ${checkboxLabelDisabled}`;
 
-export const CheckboxFormatter = forwardRef<HTMLInputElement, CheckboxFormatterProps>(
-  function CheckboxFormatter({ onChange, ...props }: CheckboxFormatterProps, ref) {
-    function handleChange(e: React.ChangeEvent<HTMLInputElement>) {
-      onChange(e.target.checked, (e.nativeEvent as MouseEvent).shiftKey);
-    }
-
-    return (
-      <label
-        className={clsx(checkboxLabelClassname, {
-          [checkboxLabelDisabledClassname]: props.disabled
-        })}
-      >
-        <input
-          type="checkbox"
-          ref={ref}
-          {...props}
-          className={checkboxInputClassname}
-          onChange={handleChange}
-        />
-        <div className={checkboxClassname} />
-      </label>
-    );
+export function checkboxFormatter(
+  { onChange, ...props }: CheckboxFormatterProps,
+  ref: React.RefObject<HTMLOrSVGElement>
+) {
+  function handleChange(e: React.ChangeEvent<HTMLInputElement>) {
+    onChange(e.target.checked, (e.nativeEvent as MouseEvent).shiftKey);
   }
-);
+
+  return (
+    <label
+      className={clsx(checkboxLabelClassname, {
+        [checkboxLabelDisabledClassname]: props.disabled
+      })}
+    >
+      <input
+        type="checkbox"
+        ref={ref as React.RefObject<HTMLInputElement>}
```

Comment:
I'm not sure how to address this type issue 🤷‍♂️

Reply:
Should we change it to `React.RefObject<HTMLInputElement>`?

Reply:
The formatter can be implemented using other elements, so that would only be good for some cases 🤷‍♂️

Reply:
It may be safe to assume checkbox formatter is always using a checkbox (may be a hidden element) for accessibility 🤔 We can revisit